### DISCUSSION TITLE
提问作者 关于“直播间右侧聊天区域宽度自定义” 的功能

### DISCUSSION THREAD

**Discussion by pencilqaq** [the1812/Bilibili-Evolved#4254] [提问 / Q&A]
#4216  中提到的功能 作者打算以何种方式实现？

1. 是作为一个单独的组件还是某个组件的功能？ 
2. 是 支持实时调节(即在页面中拖动以修改宽度)还是在组件设置中输入宽度？

如果作者还没有做这个功能 我想尝试一下

**Comment by the1812** [comment]
单独组件，输入视图比例生成宽度
